src/flyout.js

```diff
@@ -50,21 +50,12 @@ export class FlyoutElement extends LitElement {
   }
 
   _onOutsideClick = (e) => {
+    console.log("click outside");
```

Comment:
Assuming this can probably be removed, but at least should be:

```suggestion
    console.debug("Click outside");
```

Reply:
```suggestion
```

Reply:
Meh, this was just debugging 😅